content/http/dont-reimport-httpclientmodule.md

```diff
@@ -0,0 +1,16 @@
+---
+title: don't reimport HttpClientModule
+---
+
+# Problem
+
+`HttpClientModule` is responsible for providing us with HTTP related functionality. One such functionality is the ability to specify interceptors that inspect and transform HTTP requests from your application to the server.
+When reimporting `HttpClientModule` in a lazy-loaded module or a dependency of a lazy-loaded module, existing HTTP interceptors will be overridden for that module.
+
+# Solution
+
+Import `HttpClientModule` only once (in the root module).
```

Comment:
I would say we remove the brackets here, because that's an important part of the sentence. It should be part of the main sentence.

Reply:
The idea was to hint at the fact that not _every_ project will require to have it put there. However, the use-cases where this is the case are near non-existent so I can see how this would've caused confusion